python-avd/pyavd/_eos_designs/structured_config/network_services/router_adaptive_virtual_topology.py

```diff
@@ -106,7 +106,7 @@ def _cv_pathfinder_profiles(self: AvdStructuredConfigNetworkServicesProtocol) ->
             for match in policy.get("matches", []):
                 profile = {
                     "name": match["avt_profile"],
-                    "load_balance_policy": match["load_balance_policy"]["name"],
+                    "load_balance_policy": match["load_balance_policy"].name,
```

Comment:
This function is creating a hybrid of list of dicts of objects. It also uses append_if_not duplicate. I think we should refactor it to contain proper wan policy objects.

Reply:
discussed with @gmuloc and we will postpose rewrite of this.